.github/workflows/ruff.yml

```
@@ -6,3 +6,5 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: astral-sh/ruff-action@v1
+        with:
+          args: "format --check"
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)